packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/execPermissionsWalkthrough.ts

```diff
@@ -25,32 +26,31 @@ export const askExecRolePermissionsQuestions = async (
   category?: string,
```

Comment:
This value seems to be coming in as undefined for most scenarios. Is it still used?